adminStuff/viewAllTags.php

```diff
@@ -0,0 +1,40 @@
+<?php
+include('config/init.php');
+if(isset($_REQUEST['submitItem'])){
+    $errors1 = array();
```

Comment:
Making different errors arrays is a nice way to support different forms like this, although it's possible you might want to always call this the same thing so that it can be referenced in functions (like if the `singleLineTextBox` function wants to look at errors to decide if it should print an error message.

If you choose to do that, you'd need to just use one $errors array that both forms share, and then you'd need to make the field names unique (i.e. `submitTagName` vs. `deleteTagName`)